dotcom-rendering/cypress/integration/parallel-3/article.interactivity.spec.js

```diff
@@ -36,27 +36,74 @@ describe('Interactivity', function () {
 			cy.visit(`/Article?url=${articleUrl}`);
 			cy.get('[data-cy=share-counts]').should('exist');
 		});
-		it('should display all the rich links for an article', function () {
+		it('loads the discussion when you click the comment count', function () {
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/commentisfree/2022/jan/20/uk-government-yemen-war-saudi-arabia-westminster`,
+			);
+			cy.get('[data-cy=comment-counts]').should('exist');
+			// The discusion is not yet loaded
+			cy.get('[data-component=discussion]').should('not.exist');
+			// Click the comment count
+			cy.get('[data-cy=comment-counts]').click();
+			cy.get('[data-component=discussion]').should('exist');
+		});
+		it('loads the discussion immediately when you use a url ending in #comments', function () {
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/commentisfree/2022/jan/20/uk-government-yemen-war-saudi-arabia-westminster#comments`,
+			);
+			cy.get('[data-component=discussion]').should('exist');
+		});
+		// eslint-disable-next-line mocha/no-skipped-tests
+		it.skip('loads the discussion immediately when you use a permalink', function () {
+			// The permalink feature is not currently working but once it does we want this test ready to go
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/commentisfree/2022/jan/20/uk-government-yemen-war-saudi-arabia-westminster#comment-154433663`,
+			);
+			cy.get('gu-island[name=DiscussionContainer]').should(
+				'have.attr',
+				'data-gu-hydrated',
+				'true',
+			);
+			cy.get('[data-component=discussion]').should('exist');
```

Comment:
would `.should('be.visible')` be more user centric?

Reply:
Yeah, you're right, this better represents what we want to assert on